services/users/src/app.ts

```diff
@@ -44,6 +44,7 @@ app.use(
   })
 );
 app.use(express.json());
+app.use(isAuthenticated);
```

Comment:
So we can't add this here, because there are some routes on the users service that users use to login and dont require unauthentication. In this case, just add it on the route itself.